src/classes.js

```diff
@@ -2,4 +2,62 @@ class Foo {
 
 }
 
-var bar;
+var bar = new Foo();
+
+class Dog {
+  constructor(says) {
+    this.says = 'life is ruff'
+  }
+}
+var dog = new Dog;
+
+class Cat {
+
+  constructor(garfield) {
+    this.garfield = garfield
+    this.growl = function(growl) {
```

Comment:
The `growl` function should be shared across all cats. This creates a new version of `growl` for each individual cat instance. Remind me to discuss this difference in class.